src/platform/tigerlake/include/platform/lib/memory.h

```diff
@@ -249,14 +249,31 @@
 #define HEAP_SYS_RT_X_COUNT512		8
 #define HEAP_SYS_RT_X_COUNT1024		4
 
+/* Heap section counts base */
+#define HEAP_COUNT64		128
+#define HEAP_COUNT128		128
+#define HEAP_COUNT256		96
+#define HEAP_COUNT512		8
+#define HEAP_COUNT1024		4
+#define HEAP_COUNT2048		2
+#define HEAP_COUNT4096		1
+
+#if HP_SRAM_SIZE < 0x200000
+#define RT_TIMES	3
+#define RT_SHARED_TIMES	6
+#else
+#define RT_TIMES	8
+#define RT_SHARED_TIMES	16
+#endif
+
 /* Heap section sizes for module pool */
-#define HEAP_RT_COUNT64			128
-#define HEAP_RT_COUNT128		64
-#define HEAP_RT_COUNT256		128
-#define HEAP_RT_COUNT512		8
-#define HEAP_RT_COUNT1024		4
-#define HEAP_RT_COUNT2048		1
-#define HEAP_RT_COUNT4096		1
+#define HEAP_RT_COUNT64			(HEAP_COUNT64 * RT_TIMES)
```

Comment:
@keyonjie for my education, what does HEAP_COUNT64 (HEap section counts base) mean?